sale_company_currency/views/sale_order_view.xml

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+    <!-- SO -->
+    <record id="view_order_tree" model="ir.ui.view">
+      <field name="name">sale.order.tree</field>
+      <field name="model">sale.order</field>
+      <field name="inherit_id" ref="sale.view_order_tree"/>
+      <field name="arch" type="xml">
+
+        <!-- Remove multicurrencies sums (makes no sense)-->
```

Comment:
Actually it can make sense: if a company is working all sales in USD and company currency is EUR, both totals make sense (actually the one in SO currency more than the company currency).
I would rather have a filter on currency to allow to show only the SO for the given currency (and then this total makes sense when filtering)